### ISSUE TITLE
Can't quickly scroll chat-list while holding shift

### ISSUE THREAD

**Issue by crexlight** [telegramdesktop/tdesktop#27256]
### Steps to reproduce

1. Open Telegram
2. Hold Shift
3. Scroll the chat-list

### Expected behaviour

Chat-list is scrolling quickly while holding shift

### Actual behaviour

Chat-list is scrolling as if you don't hold the shift key while holding shift

### Operating system

Windows 10 22H2

### Version of Telegram Desktop

4.13

### Installation source

Static binary from official website

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/26546

**Comment by ilya-fedin** [comment]
@Aokromes no, it's not really the same. #25546 is about slow speed without scroll. Fixing the speed without shift won't fix the lack of speed change by pressing shift.

**Comment by ilya-fedin** [comment]
No need to bump, if you want this feature, feel free to implement it yourself. If you can't implement, wait patiently until someone else interested would implement it.

**Comment by ilya-fedin** [comment]
https://github.com/telegramdesktop/tdesktop/commit/cf6d13acc2e797700cf79170b5a9d2b34d69d7ab
